akka-stream/src/main/resources/reference.conf

```diff
@@ -85,6 +87,9 @@ akka {
         write-buffer-size = 16 KiB
       }
 
+      # Time to wait for async materializer creation before throwing an exception
+      creation-timeout = 20 seconds
```

Comment:
There was some subsequent fixes to extension loading because of this blocking during startup, if we make it eager we should probably backport those fixes as well.